### PR TITLE
added help via platyPS

### PR DESCRIPTION
**Pull request by aafksab**
Created initial Markdown help.
Still needs markdown files in .\docs folder updated where {{ ... }} placeholders exist with missed help content.

### CODE REVIEW

File: docs/Disable-NTFSAccessInheritance.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Disable-NTFSAccessInheritance
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```
Comment (AspenForester) [inline docs/Disable-NTFSAccessInheritance.md]: Disables inheritance of Access Control Entries from the parent directory.

File: docs/Disable-NTFSAccessInheritance.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### -RemoveInheritedAccessRules
+{{ Fill RemoveInheritedAccessRules Description }}
```
Comment (AspenForester) [inline docs/Disable-NTFSAccessInheritance.md]: Remove inherited Access Control Entries, leaving only ACEs that had already been explicitly set.

File: docs/Enable-NTFSAccessInheritance.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Enable-NTFSAccessInheritance
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```
Comment (AspenForester) [inline docs/Enable-NTFSAccessInheritance.md]: Enables inheritance of Access Control Entries from the target directory's parent.

File: docs/Enable-NTFSAccessInheritance.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### -RemoveExplicitAccessRules
+{{ Fill RemoveExplicitAccessRules Description }}
```
Comment (AspenForester) [inline docs/Enable-NTFSAccessInheritance.md]: Removes any explicit ACEs that were previously set.

File: docs/Get-ChildItem2.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Get-ChildItem2
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```
Comment (AspenForester) [inline docs/Get-ChildItem2.md]: Gets the items and child items in one or more specified locations.  Locations are not subject to the 256 character path length limit.

File: docs/Get-DiskSpace.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+## DESCRIPTION
+{{ Fill in the Description }}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
```
Comment (AspenForester) [inline docs/Get-DiskSpace.md]: PS C:\WINDOWS\system32> Get-DiskSpace c:


AvailableFreeSpacePercent  : 66.8%
AvailableFreeSpaceUnitSize : 310.14 GB
ClusterSize                : 4096
DriveName                  : c:\
TotalSizeUnitSize          : 464.3 GB
UsedSpacePercent           : 33.2%
UsedSpaceUnitSize          : 154.16 GB
FreeBytesAvailable         : 333013069824
TotalNumberOfBytes         : 498536812544
TotalNumberOfFreeBytes     : 333013069824
BytesPerSector             : 512
NumberOfFreeClusters       : 81302019
SectorsPerCluster          : 8
TotalNumberOfClusters      : 121713089

File: docs/Get-DiskSpace.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+{{ Add example description here }}
```
Comment (AspenForester) [inline docs/Get-DiskSpace.md]: Returns information about the specified local disk.

File: docs/Get-NTFSInheritance.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+```
+
+## DESCRIPTION
+{{ Fill in the Description }}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
```
Comment (AspenForester) [inline docs/Get-NTFSInheritance.md]: PS C:\WINDOWS\system32> Get-NTFSInheritance -Path \\itinfpw022\itxxhome\JAPE004

Name    AccessInheritanceEnabled AuditInheritanceEnabled
----    ------------------------ -----------------------
JAPE004 True                     True

File: docs/Get-NTFSInheritance.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Get-NTFSInheritance
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```
Comment (AspenForester) [inline docs/Get-NTFSInheritance.md]: Returns the state of Access and Audit inheritance for the specified directory

File: docs/Get-NTFSOwner.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Get-NTFSOwner
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```
Comment (AspenForester) [inline docs/Get-NTFSOwner.md]: Returns the owner information for a specified file or directory

File: docs/Get-NTFSOwner.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+```
+
+## DESCRIPTION
+{{ Fill in the Description }}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
```
Comment (AspenForester) [inline docs/Get-NTFSOwner.md]: PS C:\> Get-NTFSOwner -Path \\itinfpw022\itxxhome\JAPE004

Item                          Owner                  Account                FullName
----                          -----                  -------                --------
\\itinfpw022\itxxhome\JAPE004 BUILTIN\Administrators BUILTIN\Administrators \\itinfpw022\itxxhome\JAPE004

File: docs/Move-Item2.md [hunk 12]
```diff
@@ -0,0 +1,139 @@
...
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Move-Item2
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```
Comment (AspenForester) [inline docs/Move-Item2.md]: Move an item from one location to another.  Locations are not limited by 256 character path length.

File: docs/Remove-Item2.md [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Remove-Item2
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```
Comment (AspenForester) [inline docs/Remove-Item2.md]: Remove an item from a location. Location is not limited by 256 character path length.

File: docs/Remove-NTFSAccess.md [hunk 14]
```diff
@@ -0,0 +1,223 @@
...
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Remove-NTFSAccess
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```
Comment (AspenForester) [inline docs/Remove-NTFSAccess.md]: Remove an Access Control Entry for the specified file or directory.

File: docs/Remove-NTFSAccess.md [hunk 15]
```diff
@@ -0,0 +1,223 @@
...
+```
+
+## DESCRIPTION
+{{ Fill in the Description }}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
```
Comment (AspenForester) [inline docs/Remove-NTFSAccess.md]: PS C:\WINDOWS\system32> Remove-NTFSAccess -Path \\itinfpw022\itxxhome\JAPE004 -Account MyDomain\TrainerJami -AccessRights FullControl -AccessType Allow

File: docs/Remove-NTFSAccess.md [hunk 16]
```diff
@@ -0,0 +1,223 @@
...
+```
+
+{{ Add example description here }}
```
Comment (AspenForester) [inline docs/Remove-NTFSAccess.md]: Removes TrainerJami's FullControl ACE from the location \\itinfpw022\itxxhome\JAPE004

File: docs/Set-NTFSOwner.md [hunk 17]
```diff
@@ -0,0 +1,117 @@
...
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Set-NTFSOwner
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```
Comment (AspenForester) [inline docs/Set-NTFSOwner.md]: Takes ownership of a file or directory

File: docs/Test-Path2.md [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Test-Path2
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```
Comment (AspenForester) [inline docs/Test-Path2.md]: Determines if a ll elements of a path exist.  The path is not limited to 256 characters.

**Comment by raandree** [comment]
Thanks for submitting the PR. The help in #62 is more complete so we have merged this one.

**Pull request closed without merging**